pkg/printers/internalversion/printers.go

```diff
@@ -104,27 +107,37 @@ var (
 	podPresetColumns                 = []string{"NAME", "AGE"}
 )
 
-func printPod(pod *api.Pod, w io.Writer, options printers.PrintOptions) error {
-	if err := printPodBase(pod, w, options); err != nil {
-		return err
-	}
-
-	return nil
+func printPod(pod *api.Pod, options printers.PrintOptions) ([]metav1alpha1.TableRow, error) {
+	return printPodBase(pod, options)
 }
 
-func printPodList(podList *api.PodList, w io.Writer, options printers.PrintOptions) error {
+func printPodList(podList *api.PodList, options printers.PrintOptions) ([]metav1alpha1.TableRow, error) {
+	rows := make([]metav1alpha1.TableRow, 0, len(podList.Items))
 	for _, pod := range podList.Items {
-		if err := printPodBase(&pod, w, options); err != nil {
-			return err
+		r, err := printPodBase(&pod, options)
```

Comment:
Why does this `printPodBase` instead of `printPod`?  Seems like the general case of `FooList` is to call the row printer for `Foo` and aggregate them.